parser/src/test/org/apache/hadoop/hive/ql/parse/TestReservedWords.java

```diff
@@ -119,6 +119,7 @@ public static Collection<String> data() {
         "ONLY",
         "OR",
         "ORDER",
+        "ORDERED",
```

Comment:
I would like to include it in non-reserved words because `ORDERED` is not listed in SQL:2023.
https://github.com/apache/hive/blob/cbb14ddb1738466b08c01082470c921d51bb8046/parser/src/java/org/apache/hadoop/hive/ql/parse/IdentifiersParser.g#L931